parsl/monitoring/visualization/templates/dag.html

```diff
@@ -10,7 +10,7 @@ <h2>{{ workflow_details['workflow_name'] }}</h2>
 <ul>
     <li><b>Started:</b>  {{ workflow_details['time_began'] | timeformat }}</li>
     <li><b>Completed:</b>  {{ workflow_details['time_completed'] | timeformat }}</li>
-    <li><b>Workflow duration:</b>  {{ workflow_details['workflow_duration'] | timeformat }}</li>
+    <li><b>Workflow duration:</b>  {{ (workflow_details['time_began'] - workflow_details['time_completed']) | durationformat }}</li>
```

Comment:
It should be `(workflow_details['time_began'], workflow_details['time_completed'])`